src/wp-admin/about.php

```diff
@@ -124,8 +124,9 @@
 				<p>
 					<?php
 					printf(
-						/* translators: %s: code-formatted "data-wp-on-async", %%: escaped percent sign, leave as %%. */
-						__( '6.6 includes important updates like removing redundant WP_Theme_JSON calls, disabling autoload for large options, eliminating unnecessary polyfill dependencies, lazy loading post embeds, introducing the %s directive, and a 40%% reduction in template loading time in the editor.' ),
+						/* translators: %1$s: code-formatted "WP_Theme_JSON", %2$s: code-formatted "data-wp-on-async", %%: escaped percent sign, leave as %%. */
+						__( 'WordPress 6.6 includes important updates like removing redundant %1$s calls, disabling autoload for large options, eliminating unnecessary polyfill dependencies, lazy loading post embeds, introducing the %2$s directive, and a 40%% reduction in template loading time in the editor.' ),
```

Comment:
```suggestion
						__( 'WordPress 6.6 includes important updates like removing redundant %1$s calls, disabling autoload for large options, eliminating unnecessary polyfill dependencies, lazy loading post embeds, introducing the %2$s directive, and a 33%% reduction in template loading time in the editor.' ),
```

Sounds like the correct number is 33, see https://core.trac.wordpress.org/ticket/61320#comment:24